wordpress-mobile/WordPress-Android
# View more Posts & pages

This PR adds the refreshed View all posts & pages screen.

To test:
1. Go to Stats
2. Wait for data to load and tap on the View more button on the Posts & pages card
3. Notice that a new screen is opened with all the posts & pages for the selected period
4. Tap on the date selector and notice how the data is refreshed
